contracts/modules/Checkpoint/DividendCheckpoint.sol

```diff
@@ -136,16 +132,12 @@ contract DividendCheckpoint is DividendCheckpointStorage, ICheckpoint, Module {
         uint256 _dividendIndex,
         uint256 _start,
         uint256 _iterations
-    )
-        public
-        withPerm(DISTRIBUTE)
-        validDividendIndex(_dividendIndex)
-    {
+    ) public withPerm(DISTRIBUTE) validDividendIndex(_dividendIndex) {
         Dividend storage dividend = dividends[_dividendIndex];
         address[] memory investors = ISecurityToken(securityToken).getInvestors();
         uint256 numberInvestors = Math.min(investors.length, _start.add(_iterations));
         for (uint256 i = _start; i < numberInvestors; i++) {
-            address payee = investors[i];
+            address payable payee = address(uint160(investors[i]));

```

Comment:
Can't we do cast directly with `address payable payee = investors[i];`?

Reply:
unfortunately not :(. I ranted about this in slack :p.